main.ipynb

```diff
@@ -444,16 +452,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 12,
    "id": "f7f9334a-6ee1-4aa4-8770-d0f48b36f18b",
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Baseline dataset_spec: DatasetSpec(overlap=0.7, sidelap=0.7, height=30.48m, scan_area=150x150m, exposure=2ms)\n",
+      "Baseline distances (m) [dx, dy]: [15.16791291 11.38070491]\n",
+      "\n",
+      "Double height\n",
+      " distances: [30.33582583 22.76140983]  ratio to baseline: [2. 2.]\n",
+      "Distance increase factor (should be ~2x): [2. 2.]\n",
+      "\n",
+      "Halve focal length (fx,fy * 0.5)\n",
+      " distances: [30.33582583 22.76140983]  ratio to baseline: [2. 2.]\n",
+      "Distance increase factor (should be ~2x): [2. 2.]\n",
```

Comment:
Can you add a little bit of intuition behind the results and/or when do you expect pilots to modify settings similar to your experiment?

Also, I would recommend you to use markdown cells for the text.